docs/source/getting-started/quickstart-2/index.rst

```diff
@@ -220,13 +247,15 @@ Transition the model to **Staging** by choosing the **Stage** dropdown:
 Serve the model locally
 ----------------------------
 
-MLflow allows you to easily serve models produced by any run or model version. You can serve the model you just registered by running:
+MLflow allows you to easily serve models produced by any run or model version. You can serve the model
+you just registered by running:
 
 .. code-block:: bash
 
   mlflow models serve -m "models:/wine-quality/Staging" --port 5002
 
-(Note that specifying the port as above will be necessary if you are running the tracking server on the same machine at the default port of **5000**.)
+(Note that specifying the port as above will be necessary if you are running the tracking server on the
```

Comment:
We might want to call out that we DO NOT recommend serving a model from a remote tracking server's VM. ;) This is only mentioned here because we're doing a local demonstration from our own computer. In production, a serving VM should not have resource contention to deal with by running both a tracking server and an inference server on the same hardware.